sr_hardware_interface/include/sr_hardware_interface/tactile_sensors.hpp

```diff
@@ -90,7 +93,7 @@ class GenericTactileData
     {
       // Convert Git revision to hexadecimal long hash
       std::stringstream git_revision;
-      for (int i = 2; i < 22; i++)
+      for (int i = 2; i < 23; i++)
```

Comment:
Will the string have length 23 for all the types of sensors now?
Remember that this class is used for all the versions of the ehtercat protocol.
So it should work well with any of the possible template arguments of the driver class.